src/coreclr/nativeaot/System.Private.CoreLib/src/System/RuntimeType.cs

```diff
@@ -120,5 +120,9 @@ public sealed override Array GetEnumValuesAsUnderlyingType()
 
         internal bool IsActualEnum
             => TryGetEEType(out EETypePtr eeType) && eeType.IsEnum;
+
+#pragma warning disable IDE0060
+        internal static object AllocateValueType(RuntimeType type, object? value) => throw new NotSupportedException();
```

Comment:
Why does this throw NotSupportedException? Comment would be useful.

Reply:
Removed; this was an early attempt to share some code.